image/rhel/Dockerfile.envsubst

```
@@ -43,7 +44,7 @@ RUN ln -s entrypoint-wrapper.sh /stackrox/admission-control && \
     ln -s entrypoint-wrapper.sh /stackrox/compliance && \
     ln -s entrypoint-wrapper.sh /stackrox/kubernetes-sensor && \
     ln -s entrypoint-wrapper.sh /stackrox/sensor-upgrader && \
-    ln -s /assets/downloads/cli/roxctl-linux-amd64 /stackrox/roxctl && \
+    ln -s /assets/downloads/cli/roxctl-linux-${TARGET_ARCH} /stackrox/roxctl && \
```

While you're at it, you could also add
```suggestion
    ln -s /assets/downloads/cli/roxctl-linux-${TARGET_ARCH} /stackrox/roxctl && \
    ln -s /assets/downloads/cli/roxctl-linux-${TARGET_ARCH} /stackrox/roxctl-linux && \
```

to fix an issue `infra` is observing (see https://srox.slack.com/archives/CELUQKESC/p1662669354128059 for details).

I'm on the fence on whether it is a good idea to combine these two, but the change suggested above will have to be cherry-picked, and it seems fairly conflict-prone with your change so I'd rather keep it in one commit/PR. WDYT?

---

Malte is addressing this in a separate PR then I will rebase as that will ensure a clean cherry pick onto release branch